docs/content/docs/shell-config/keybindings.md

```diff
@@ -20,7 +20,7 @@ when `Control+L` is pressed. How keybindings are represented is a bit more of an
 internal detail and not very fun to write, so a macro is provided to make the
 experience a bit better.
 ```rust
-ley keybinding = keybindings! {
+let keybinding = keybindings! {
     "C-l" => Command::new("clear").spawn(),
```

Comment:
should update docs to reflect that a description needs to be passed now

Reply:
I updated the docs. Here is a screenshot of what they look like now:

![shrs3](https://github.com/MrPicklePinosaur/shrs/assets/47926823/b1311ecd-262a-465c-9663-4c76c2926e8f)